chp00_introduction/NOC_I_03_RandomWalkTendsToRight/sketch.js

```diff
@@ -15,18 +15,20 @@ function draw() {
   walker.render();
 }
 
-function Walker() {
-  this.x = width/2;
+class Walker{
+  constructor(){
+    this.x = width/2;
     this.y = height/2;
+  };
 
-  this.render = function() {
+ render() {
     stroke(0);
     point(this.x,this.y);
   };
 
-  this.step = function() {
-    var choice = floor(random(4));
-    var r = random(1);
+  step(){
+    const choice = floor(random(4));
```

Comment:
I'm not sure about using `const`, it may be more technically correct but I'm a bit worried about staying beginner friendly, what do you think?

Reply:
I was using lebab as a first-pass transpiler and they recommend const. I agree that it may be unnecessarily technical. Would you use let or var instead?

Reply:
let!